uber/templates/registration/form.html

```diff
@@ -266,7 +266,10 @@
         <label class="col-sm-2 control-label">Checked In</label>
         <div class="col-sm-6">
             {% if attendee.checked_in %}
-                Checked in at {{ attendee.checked_in_local }}
+                Checked in at {{ attendee.checked_in_local }}{% if c.HAS_ACCOUNTS_ACCESS %}<br>
+                <i>Was this an error?</i>
+                <input type="button" value="Check-Out" onclick="confirmAction()">
```

Comment:
Couple of style notes:

1) ``confirmAction`` is not a good name for this Javascript function, because it's super-generic and doesn't actually describe what the function is doing.  ``undoCheckin`` would be a much better name.

2) Similarly, ``Check-Out`` might be misleading/confusing as a button name, and ``Undo Checkin`` would probably be better.